content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -50,5 +51,57 @@ ActivityFeed.propTypes = {
   length: React.PropTypes.number
 };
 
+function groupSitesByDate(sites) {
+  let groupedSites = new Map();
+  for (let site of sites) {
+    if (!Number.isInteger(site.lastVisitDate)) {
+      continue;
+    }
+
+    let day = moment(site.lastVisitDate / 1000).startOf("day").format();
+    if (!groupedSites.has(day)) {
+      groupedSites.set(day, []);
+    }
+    groupedSites.get(day).push(site);
+  }
+  return groupedSites;
+}
+
+const GroupedActivityFeed = React.createClass({
+  getDefaultProps() {
+    return {length: DEFAULT_LENGTH};
+  },
+  render() {
+    const sites = this.props.sites.slice(0, this.props.length);
+    const groupedSites = groupSitesByDate(sites);
+    return (<div className="grouped-activity-feed">
+      {this.props.title &&
```

Comment:
Should we add a `title` to the `getDefaultProps()` section above so we always have a default section title?